test/unit/controllers/DeviceController.test.js

```diff
@@ -47,5 +49,35 @@ describe('DeviceController', () => {
         })
       })
     })
+
+    it('switches current device to new user', () => {
+      _.extend(req.params, {
+        token: deviceToken,
+        platform: platform
+      })
+
+      req.headers = {'ios-version': version}
+
+      return Device.fetchAll()
+      .then(devices => Promise.map(devices.models, device => device.destroy()))
+      .then(() => Device.forge({
+        token: req.param('token'),
+        platform: platform,
+        version: version,
+        user_id: fixtures.u2.id
+      })
+      .save()
```

Comment:
nitpick: this should be on the line above, right after `})`. when it's on its own line the indentation suggests its outside of the `.then` above it, but it's actually still nested inside it.

likewise, i think you want to add another `)` after `.save()` and remove it from line 80 so that the indentation below makes sense